### PR TITLE
Add unregister.

### PR DESCRIPTION
**Pull request by Nonsense331**

### CODE REVIEW

File: src/main/java/com/cloudfiveapp/push/CloudFivePush.java [hunk 1]
```diff
@@ -308,7 +338,47 @@ private void notifyCloudFive() {
...
+                version = applicationContext.getPackageManager().getPackageInfo(applicationContext.getPackageName(), 0).versionName;
+            } catch (PackageManager.NameNotFoundException e) {
+                version = "unknown";
+            }
+            nameValuePairs.add(new BasicNameValuePair("app_version", version));
+            httppost.setEntity(new UrlEncodedFormEntity(nameValuePairs));
+            HttpResponse response = httpclient.execute(httppost);
+        } catch (ClientProtocolException e) {
+            // TODO Auto-generated catch block
+        } catch (IOException e) {
```
Comment (petitJAM) [inline src/main/java/com/cloudfiveapp/push/CloudFivePush.java]: You should combine these catch blocks so the logging still happens `catch (ClientProtocolException | IOException e) {`

File: src/main/java/com/cloudfiveapp/push/CloudFivePush.java [hunk 2]
```diff
@@ -153,6 +162,27 @@ protected Object doInBackground(Object[] params) {
         }
     }
 
+    private void unregisterForRemoteNotifications() {
+        if (checkPlayServices()) {
+            gcm = GoogleCloudMessaging.getInstance(applicationContext);
+            registrationId = getRegistrationId();
+
+            if (registrationId.isEmpty()) {
+                // registerInBackground();
```
Comment (petitJAM) [inline src/main/java/com/cloudfiveapp/push/CloudFivePush.java]: what's this

**Comment by petitJAM** [comment]
https://rollbar.com/tenforward/habit_catalyst-android/items/24/

`GCMIntentService` was using `Notification.Builder` and calling `build()` on it which doesn't exist in SDK level 15

**Comment by samsonasu** [comment]
Why check in the .idea files? Also ship it

**Comment by petitJAM** [comment]
those are just the .idea files that are supposed to be checked in. idk exactly what they do

**Pull request merged**